client/http_sender.go

```diff
@@ -123,22 +120,20 @@ func (s *httpSender) post(call proto.Call) error {
 
 		resp, err := s.client.Do(req)
 		if err != nil {
-			return retry.Continue, err
+			continue
```

Comment:
I think that in general when we `continue` after an error we should log it. There may be times where that is too noisy, but our default habit should be to log something.